deployment/puppet/refinery/manifests/python.pp

```diff
@@ -7,16 +7,21 @@
   $django_root         = $refinery::params::django_root,
 ) inherits refinery::params {
 
+
   class { '::python':
+    version    => 'python3.5',
```

Comment:
the ubuntu version we're on comes with 3.5 - otherwise we'd have to pull from [here](https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa)  also, i don't know if django 1.8 supports 3.8 officially - the website was somewhat unclear.

Reply:
Good point, Django 1.8 - 1.10 support only up to 3.5 (https://docs.djangoproject.com/en/1.11/faq/install/#what-python-version-can-i-use-with-django)

However, 3.5 end-of-life is 2020-09-13 (https://devguide.python.org/#status-of-python-branches) - that should motivate upgrade to Ubuntu 18.04 (it comes with 3.6) and to Django 1.11+

Reply:
yup, i filed that away in the back of my head while doing this :)